example/cumulus-tf/wait_for_deploy_workflow.tf

```diff
@@ -0,0 +1,40 @@
+module "wait_for_deploy_workflow" {
+  source = "../../tf-modules/workflow"
+
+  prefix          = var.prefix
+  name            = "WaitForDeployWorkflow"
+  workflow_config = module.cumulus.workflow_config
+  system_bucket   = var.system_bucket
+  tags            = local.default_tags
+
+  state_machine_definition = <<JSON
+{
+  "Comment": "Tests redeployment during workflow run",
+  "StartAt": "HelloWorld",
+  "States": {
+    "HelloWorld": {
+      "Type": "Task",
+      "Resource": "${module.cumulus.hello_world_task.task_arn}",
+      "Next": "WaitForS3ObjectToExist"
+    },
+    "WaitForS3ObjectToExist": {
```

Comment:
Now using a "WaitForS3ObjectToExist" task instead of a "Wait for CloudFormation to finish" task.